packages/turbo/android/src/main/java/com/reactnativeturbowebview/RNTurboError.kt

```diff
@@ -0,0 +1,50 @@
+package com.reactnativeturbowebview
+
+import android.webkit.WebViewClient
+
+enum class RNTurboError(val code: Int) {
+  HTTP(1),
+  NETWORK_FAILURE(0),
```

Comment:
Just out of curiosity, I wonder if you really want to have `0` as an error code, since in general (and in POSIX) `0` means success.

Reply:
Unfortunately `turbo-ios` defines code 0 as a network failure so to match both native implementations I had to use this code (see [this](https://github.com/hotwired/turbo-ios/blob/c9d060fcdec136c3808cd0764e956d375012c6fb/Source/TurboError.swift#L4))

Reply:
😢